### PR TITLE
Adds Wordpress example

### LINKED ISSUE #57: Migrate WordPress examples

### PR DESCRIPTION
**Pull request by LeendersR**
Adds the Wordpress example to resolve #57. Also includes a new version of replicatedservice.py that accepts three new properties:

cluster_ip: to set the ip of a service
privileged: to create privileged containers
volumes: to mount volumes

### CODE REVIEW

**Comment by LeendersR** [comment]
@bmelville This should work.

File: templates/replicatedservice/v2/replicatedservice.py.schema [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    type: array
+    description: Volumes to mount.
+    items:
+      type: object
+      properties:
+        mounth_path:
+          type: string
+          description: Path to mount volume
+        # See https://cloud.google.com/container-engine/docs/spec-schema?hl=en for possible volumes. Since we only use gcePersistentDisk and NFS in our examples we have only added these ones.
+        oneOf:
```
Comment (bmelville) [inline templates/replicatedservice/v2/replicatedservice.py.schema]: Cool, this is a great example to have of advanced schema stuff. Thanks!

File: templates/nfs/v1/nfs.jinja [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set PROPERTIES = properties or {} %}
+{% set SERVER_IP = PROPERTIES['ip'] or '10.0.253.247' %}
+{% set SERVER_PORT = PROPERTIES['port'] or 2049 %}
+{% set SERVER_DISK = PROPERTIES['disk'] or 'nfs-disk' %}
+{% set SERVER_DISK_FSTYPE = PROPERTIES['fstype'] or 'ext4' %}
+{% set CLAIM_NAME = PROPERTIES['claim-name'] or 'nfs' %}
+
+resources:
+- name: nfs
+  type: https://raw.githubusercontent.com/leendersr/deployment-manager/master/templates/replicatedservice/v2/replicatedservice.py
```
Comment (zmerlynn) [inline templates/nfs/v1/nfs.jinja]: Is this right? I think it needs to be `type: https://raw.githubusercontent.com/kubernetes/deployment-manager/master/templates/replicatedservice/v2/replicatedservice.py` like rest (as this no longer seems to resolve).

**Comment by LeendersR** [comment]
Good catch, thanks.

**Pull request merged**